CHANGELOG.md

```diff
@@ -20,6 +20,8 @@ This is the log of notable changes to the Expo client that are developer-facing.
 ### 🐛 Bug fixes
 
 - fix app cache size blowing up when using `ImagePicker` by [@sjchmiela](https://github.com/sjchmiela) ([#2750](https://github.com/expo/expo/pull/2750))
+- [iOS] Relaxed file system permissions when in Expokit by [@Szymon20000](https://github.com/Szymon20000)
+([#2808](https://github.com/expo/expo/pull/2808))
```

Comment:
Remove this entry here and add the `[#2808](https://github.com/expo/expo/pull/2808)` part to already present:
> fix FileSystem forbidding access to external directories by @Szymon20000 (#2748)